### PR TITLE
Prevent analytics clients from notifying via Airbrake

### PR DESCRIPTION
**Pull request by ceslami**
Addressing: https://github.com/Betterment/test_track_rails_client/issues/9

Failures in analytics clients tend to be temporary runtime/network
failures and not an indication that there is broken code to
fix. As a remedy, we *only* log in the default error_handler
proc in SafeWrapper, which wraps all analytics clients.

/domain @samandmoore @jmileham @aburgel @rynonl

### CODE REVIEW

File: spec/dummy/tmp/test_track_schema.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- b
+- c
+splits:
+  balance_unit:
+    dollar: 50
+    pound: 25
+    doge: 25
+  blue_button:
+    'false': 50
+    'true': 50
```
Comment (ceslami) [inline spec/dummy/tmp/test_track_schema.yml]: Hmm -- maybe we don't want to commit this?

**Comment by rynonl** [comment]
<< domainlgtm

File: spec/dummy/tmp/test_track_schema.yml (hunk 1 above)
Comment (rynonl) [inline spec/dummy/tmp/test_track_schema.yml]: oh hrm, browsed right over this.  yea maybe gitignore everything but the .keep file.

**Comment by ceslami** [comment]
There's also the `spec/dummy/tmp/cache` directory, which if not present, breaks the tests. After I run `bundle exec rake`, it empties that directory, including the `.keep` file, so it gets left out. Without that directory, a clean pull of this branch will fail until the developer adds it back.

Any ideas? Couldn't find the code/config that explicitly empties that directory out after `rake` runs.

**Comment by rynonl** [comment]
there's a `mkdir spec/dummy/cache` in the `before` block of the travis config. Seems reasonable to add(or move) that to a `before :spec` hook in the rake file

File: .gitignore [hunk 2]
```diff
@@ -5,7 +5,7 @@ pkg/
 spec/dummy/db/*.sqlite3
 spec/dummy/db/*.sqlite3-journal
 spec/dummy/log/*.log
-spec/dummy/tmp/
```
Comment (jmileham) [inline .gitignore]: you want the `spec/dummy/tmp/` (in fact, it'd be appropriate to prefix all these spec lines with a `/` - you then want to explicitly allow .keep with a `.keep` line.

**Comment by jmileham** [comment]
/platform @jmileham
<< platform

File: .gitignore (hunk 2 above)
Comment (jmileham) [inline .gitignore]: See https://github.com/Betterment/test_track_chrome_extension/pull/5/files

**Comment by jmileham** [comment]
tafn - just want to get the `.gitignore`ing right.

File: .gitignore (hunk 2 above)
Comment (jmileham) [inline .gitignore]: This still needs to be in here

**Comment by jmileham** [comment]
platform lgtm!

**Comment by ceslami** [comment]
Ha! Beat me to the bump!

**Comment by ceslami** [comment]
Thanks for your help with the cache clearing issue @jmileham 

Nice outcome of this: contributors will have a much better first-time experience!

**Pull request merged**